### PR TITLE
Add custom path for bundled files

### PR DESCRIPTION
**Pull request by johnvuko**
I add 2 options to your program for be able to define where the "bunlded" files will be. By defaut you set the files in a folder "bundled", now you can define any folder relative to the root directory. Also i add some trivial test.

### CODE REVIEW

**Comment by GoalSmashers** [comment]
@jonathantribouharet thanks for a valuable addition! There will be a few minor tweaks and we are ready to go!

File: Readme.md [hunk 1]
```diff
@@ -105,13 +105,9 @@ First, install dependencies
...
 
-Then, run the specs (on *nix):
+Then, run the specs:
 
-    make test
-
-or on Windows:
-
-    .\test.bat
+    npm test
```
Comment (GoalSmashers) [inline Readme.md:110]: Nice, thanks for getting rid of it!

File: bin/assetspkg [hunk 2]
```diff
@@ -31,6 +31,11 @@ var options = {
   root: path.join(process.cwd(), argv.r || argv.root || 'public')
 };
 
+options.bundledPath = {
+    javascripts: argv.bj || argv['js-bundled'] || path.join(options.path.javascripts, '/bundled'),
+    stylesheets: argv.bs || argv['styles-bundled'] || path.join(options.path.stylesheets, '/bundled')
+};
+
```
Comment (GoalSmashers) [inline bin/assetspkg:38]: All good, but please indent using 2 spaces.

**Comment by johnvuko** [comment]
Done for the indent.

**Comment by GoalSmashers** [comment]
@jonathantribouharet great - can you squash your last commit with `Add bundledPath options` one to keep the history cleaner? Once done we will merge your PR.

**Comment by johnvuko** [comment]
Ok good.

**Comment by GoalSmashers** [comment]
@jonathantribouharet It's merged! Let's see what else we could add to release a new version of assets-packager - both #14 and #15 do not seem to be very complex.

We'll update you tomorrow!

**Comment by johnvuko** [comment]
Ok i take care of #15.

**Comment by GoalSmashers** [comment]
Cool. Much appreciated!

**Pull request merged**